lib/bundle/mac_app_store_installer.rb

```diff
@@ -0,0 +1,21 @@
+module Bundle
+  class MacAppStoreInstaller
+    def self.install(name, id)
```

Comment:
You could see git history on how we auto install brew-cask(when it used to be a formula)